web_external/InfoView/style.styl

```diff
@@ -7,26 +7,29 @@
     flex-direction column
 
   .panel-body
-    flex 1
-    position relative
+    flex 1 1 0
+    display flex
+    flex-direction column
+    overflow-y hidden
 
   ul
+    flex 1
+    overflow-y auto
+
     &.geometry
-      padding-left 20px
+      padding-left 1em
```

Comment:
@mwoehlke-kitware lets not change px to em for now until we have a follow up discussions.